socorro/cron/jobs/fetch_adi_from_hive.py

```diff
@@ -238,7 +241,7 @@ class FetchADIFromHiveCronApp(BaseCronApp):
     @staticmethod
     def remove_control_characters(s):
         if isinstance(s, str):
-            s = unicode(s, 'utf-8', errors='replace')
+            s = text_type(s, 'utf-8', errors='replace')
```

Comment:
Similar to the change in `scripts/hive_adi_test.py` this code is mostly dead. Technically, code like it runs on a server in a data center somewhere, but that code is maintained by someone logging into the server and "fixing" it live--it's not maintained by making changes in this repository. I copied the most recent version of the file here just so I had a copy of it somewhere if that server died.

Fun fact--that server is running Python 2.6--it's not even running Python 2.7. It will never run Python 3. So given all that, any change we make here is probably fine, too. If I have to go fix the code running on that server, I'll take the obligation to make changes that work with linting here as well as on the server there.